articles/tokens/access_token-vs-id_token.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Auth0 access_token vs id_token
+---
+
+# Auth0 access_token vs id_token
+
+## When should you use the `access_token` and the `id_token`?
+
+One of the common questions that gets asked is when you should use the `access_token` and when you should use the `id_token`? 
+
+The answer is as follows:
+
+* If you need to pass a user's credentials to your own API in order to authenticate the user against the API, then you should use the `access_token`
+* When you want to get information about a user from the [/tokeninfo endpoint](/auth-api#post--tokeninfo), then you should use the `id_token`
```

Comment:
@jerriep: I question whether someone would even call the `/tokeninfo` endpoint. If they have an `id_token` it should have been scoped to contain all of the user profile claims their app needs. If they need more than this, then they should use the `access_token` to call the `/userinfo` endpoint to get the _full_ profile.

What are your thoughts @jaredhanson ?